provisioning_client/adapters/hsm_client_riot.c

```diff
@@ -155,33 +136,39 @@ static const HSM_CLIENT_X509_INTERFACE x509_interface =
     hsm_client_riot_get_common_name
 };
 
-static int generate_root_ca_info(HSM_CLIENT_X509_INFO* riot_info, RIOT_ECC_SIGNATURE* tbs_sig)
+// Free the mbedtls_mpi members of the signature
+static void ecc_signature_destroy(RIOT_ECC_SIGNATURE* tbs_sig);
```

Comment:
Maybe move the other function up here? It's just two lines of code and this is adding 2 more.

Reply:
Fixed

Reply:
Fixed